builder/dockerfile/parser/testfiles/escape_whitespace/Dockerfile

```diff
@@ -0,0 +1,5 @@
+  #escape = `
+
+FROM image
+ENV foo `
+ENV bar baz
```

Comment:
github shows some funky chars at the end - is that due to a missing crlf?  Was that on purpose and part of the test? if so, let's add a comment so that people don't edit the file and add it back in by mistake.